modules/lang-painless/src/main/antlr/PainlessParser.g4

```diff
@@ -156,11 +156,11 @@ postdot
     ;
 
 callinvoke
-    : DOT DOTID arguments
+    : nullSafe=COND? DOT DOTID arguments
```

Comment:
The way we've been doing this is simply to check to see if the Token is null, so the nullSafe is necessary.  You can just do `ctx.COND() != null` in the Walker.  This keeps the grammar a bit cleaner IMO (unless I missed something?)